tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -6,4 +6,6 @@ def __init__(self, *args, **kwargs):
         super(ImportCluster, self).__init__(*args, **kwargs)
 
     def run(self):
+        self.parameters['Node.fqdn'] = NS.node_context.fqdn
```

Comment:
Do we need this to be set in params? Can we use `NS.node_context.fqdn` where we need?